PackageConfig.cmake

```diff
@@ -384,9 +387,14 @@ configure_file(
 )
 
 # create and install ProjectConfigVersion.cmake
-write_basic_package_version_file(
-  ${PROJECT_BINARY_DIR}/pkg/${PROJECT_NAME}ConfigVersion.cmake
-  VERSION ${VERSION_MAJOR}.${VERSION_MINOR} COMPATIBILITY SameMajorVersion)
+if(${UPPER_PROJECT_NAME}_CONFIG_VERSION_IN)
+  set(PACKAGE_VERSION "${VERSION_MAJOR}.${VERSION_MINOR}")
```

Comment:
This is confusing with CMAKE_PACKAGE_VERSION used in other places. Can this be delegated to the .in file or prefixed with CVF_ or PCV_?